py_configuration_files/lms.py

```diff
@@ -554,6 +554,12 @@ def should_show_debug_toolbar(request):  # lint-amnesty, pylint: disable=missing
     'http://localhost:1996',  # frontend-app-learner-dashboard
 ]
 
+############################ Codejail ############################
+
+# Disabled by default since codejail service needs to be configured
+# and started separately. See docs/codejail.rst for details.
+ENABLE_CODEJAIL_REST_SERVICE = False
+CODE_JAIL_REST_SERVICE_HOST = "http://edx.devstack.codejail:8080"
```

Comment:
Author's note: I had thought the gunicorn port wouldn't matter and I could just set it to a conventional value, but I forgot it would be used by inter-service communication in devstack. This makes for an odd mismatch with the rest of the ports seen in devstack. Maybe it's fine, though.

Reply:
You could leave a code comment:
```
The port does not match the pattern used for other devstack services because...
```

Reply:
@timmc-edx: Again, if you thought the note https://github.com/edx/devstack/pull/109#discussion_r1953596473 was worthwhile in the PR, why not add a code comment that helps clarify?

Reply:
Oh yeah, forgot about this one... I was trying to figure out if there was a better place to document it, but I'll just add a comment.